required/amsmath/amsmath.dtx

```diff
@@ -76,17 +76,15 @@ Bug reports can be opened (category \texttt{#1}) at\\%
 [1994/12/01]% LaTeX date must be December 1994 or later
 %    \end{macrocode}
 %
-%    Providing a rollback to earlier version(s)
+%    Providing a rollback to earlier version(s). Anything prior to
+%    2018 we attempt to fulfill by using the version from 2018.
 %    \begin{macrocode}
-\providecommand\DeclareRelease[3]{}
-\providecommand\DeclareCurrentRelease[2]{}
-%
-\DeclareRelease{}{2018-12-01}{amsmath-2018-12-01.sty}
+\DeclareRelease{}{1994-06-01}{amsmath-2018-12-01.sty}
```

Comment:
For `textcomp` you have argued (https://github.com/latex3/latex2e/pull/1335#discussion_r1577494948) that it is better to keep the line `\DeclareRelease{}{2018-08-11}{textcomp-2018-08-11.sty}`, even though it is not technically necessary, so that you have

```latex
\DeclareRelease{}{1997-12-01}{textcomp-2018-08-11.sty}
\DeclareRelease{}{2018-08-11}{textcomp-2018-08-11.sty}
```

Is there a reason why this is not applied uniformly? E. g. for `amsmath`

```latex
\DeclareRelease{}{1994-06-01}{amsmath-2018-12-01.sty}
\DeclareRelease{}{2018-12-01}{amsmath-2018-12-01.sty}
```